getting-started/singleframe.rst

```diff
@@ -0,0 +1,164 @@
+These tutorials suggest that you put the outputs in collections under a namespace defined by your username since that is unique for a given system.
+In this case, there is little reason to be so careful because you are likely to have cloned into a space not shared with others.
+However, it is good practice for times when you may be using a repository with a registry used by other users on the same system.
+
+The ``--register-dataset-types`` switch tells the butler to register a dataset type if it doesn't already have a definition for it.
+Because pipelines are allowed to define datasets at runtime, this switch is necessary if you expect products to be produced that are not already represented in the registry as in this case where we are producing calibrated exposures in a repository that contains only ``raw`` files.
+If you expect that all of the dataset types should already be registiered, as is the case when processing another subset of data with a pipeline that has already been run, it can help catch unexpected behavior to remove that switch.
+
+.. tip::
+   It is not included in the above command, but the ``-j`` option is useful if you have more than one core available to you.
+   Specifying ``-j #`` will run in parallel where ``#`` is the number of processes to execute in parallel.
+
+   :ref:`Dataset queries <daf_butler_queries>` can be specified using the ``-d`` argument to specify which specific datasets should be considered when building the execution graph.
+   If this argument is omitted, all data in the repository that can be processed based on other inputs, e.g. calibrations, will be.
+
+Aside: collections and quantum graphs
+============================================
+
+:ref:`Collections <daf_butler_organizing_datasets>` are the primary way data in butler repositories are organized.
+Of the types of collections available, the two of interest here are the ``RUN`` and ``CHAINED`` types.
+
+``RUN`` collections are the least flexible.
+Once a dataset is added to a ``RUN`` collection, it can never be moved to a different ``RUN`` collection.
+The constraints on datasets in ``RUN`` collections makes these collections that most efficient to store and query.
+The collection containing the raw data is a ``RUN`` collection.
+
+``CHAINED`` collections are groupings of other collections associated with an alias for that grouping.
+The grouping of collections defines the order of collections to search when looking for a dataset associated with a specific data ID.
+The collection produced from the ``-o`` option above is a ``CHAINED`` collection.
+The ouput collection will general include all the collections in the input plus any ``RUN`` collections produced by the processing.
+
+The first step of process data is to produce the quantum graph for the processing.
+This is a directed acyclic graph that completely defines inputs and outputs for every node (quantum) in the graph.
```

Comment:
In reply to a comment I can no longer see, you *could* show the visualized graph for the simple case of a single visit+couple of detectors -- that would let you also show that you can run `pipetask qgraph` in advance to get an idea of what is going to happen without doing the processing itself.

Reply:
OK.  I believe I did that with a PNG shipped with the repo.  Let me know if that's not what you intended